tests/test_java.py

```diff
@@ -156,3 +157,30 @@ def test_java_async_profiler_musl_and_cpu(
         assert_function_in_collapsed(
             "do_syscall_64_[k]", "java", process_collapsed, True
         )  # ensure kernels stacks exist
+
+
+@pytest.mark.parametrize("in_container", [False])
```

Comment:
Also test the container flow? (tests the NS pid extraction)

Reply:
And also - test ErrorFile?

Reply:
Added